### PR TITLE
JAVA-10083 Create new module hibernate-queries from spring-hibernate-5 module

### PR DESCRIPTION
**Pull request by anuragkumawat**

### CODE REVIEW

File: persistence-modules/hibernate-queries/pom.xml [hunk 1]
```diff
@@ -49,9 +81,34 @@
             <artifactId>jmh-generator-annprocess</artifactId>
             <version>${jmh-generator.version}</version>
         </dependency>
-    </dependencies>
+        
+        <dependency>
```
Comment (kwoyke) [inline persistence-modules/hibernate-queries/pom.xml:85]: Do we really need these jaxb* dependencies?

File: persistence-modules/hibernate-queries/pom.xml [hunk 2]
```diff
@@ -49,9 +81,34 @@
...
+    </dependencies> 
 
     <properties>
+   		<!-- Spring -->
+        <org.springframework.version>5.0.2.RELEASE</org.springframework.version>
+        <org.springframework.data.version>1.10.6.RELEASE</org.springframework.data.version>
+        <!-- persistence -->
+        <hibernate.version>5.2.10.Final</hibernate.version>
+        <tomcat-dbcp.version>9.0.0.M26</tomcat-dbcp.version>
+        <hsqldb.version>2.3.4</hsqldb.version>
```
Comment (kwoyke) [inline persistence-modules/hibernate-queries/pom.xml:109]: Where is the hsqldb.version being used?

File: persistence-modules/hibernate-queries/pom.xml [hunk 3]
```diff
@@ -49,9 +81,34 @@
...
+			 <version>${com.sun.xml.version}</version>
+		</dependency>
+    </dependencies> 
 
     <properties>
+   		<!-- Spring -->
+        <org.springframework.version>5.0.2.RELEASE</org.springframework.version>
+        <org.springframework.data.version>1.10.6.RELEASE</org.springframework.data.version>
+        <!-- persistence -->
+        <hibernate.version>5.2.10.Final</hibernate.version>
```
Comment (kwoyke) [inline persistence-modules/hibernate-queries/pom.xml:107]: By providing the hibernate.version explicitly here you overwrite the one provided in the parent pom.xml and actually downgrade it. Is it intended?

File: persistence-modules/hibernate-queries/pom.xml (hunk 1 above)
Comment (anuragkumawat) [reply persistence-modules/hibernate-queries/pom.xml]: Unused jaxb* dependencies have been removed. The comment has been addressed.

File: persistence-modules/hibernate-queries/pom.xml (hunk 3 above)
Comment (anuragkumawat) [reply persistence-modules/hibernate-queries/pom.xml]: Hibernate.version has been removed. The comment has been addressed.

File: persistence-modules/hibernate-queries/pom.xml (hunk 2 above)
Comment (anuragkumawat) [reply persistence-modules/hibernate-queries/pom.xml]: Unused hsqldb.version has been removed. The comment has been addressed.

**Pull request merged**